src/utils/hooks/useNotifyProjects.ts

```diff
@@ -20,7 +22,11 @@ const useNotifyProjects = () => {
       if (filterAppDomain) {
         explorerUrl.searchParams.set('appDomain', filterAppDomain)
       } else {
-        explorerUrl.searchParams.set('is_verified', isDevModeEnabled ? 'false' : 'true')
+        explorerUrl.searchParams.set('isVerified', isDevModeEnabled ? 'false' : 'true')
+
+	if(USE_CURATED_PROJECTS) {
```

Comment:
Spacing off in some places throughout + mixing tabs with spaces:

```suggestion
	if (USE_CURATED_PROJECTS) {
```